### PR TITLE
Remove 'Create Game' and include as part of round.

### PR DESCRIPTION
**Pull request by ShawnSidoruk**
Primarily this clarifies the create game endpoint.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -451,14 +451,19 @@ curl \
...
+  "game_label": "Starburst"
 }' https://ns-api-staging.betable.com/lfs/v100\
 /users/QEaakM0qumTJd5tj/rounds
 ```
 `POST /users/{user_id}/rounds`
 
 This method creates a round. A round is a container used for grouping together wagers and payouts that are associated to a user. Any combination of wagers and payouts can be contained within a round, as long as the round has not been closed.
 
-One round may share many wager and payout events as long as the game_id and user_id remain the same.
+A round can consist of zero or more Wagers, and zero or more Payouts. The choice of how many Wagers and Payouts to include in a round is up to the integration provider.
```
Comment (NolanK) [inline source/index.html.md]:
```suggestion
A round can consist of zero or more Wagers, and zero or more Payouts. The choice of how many Wagers and Payouts to include in a round is up to the integration partner.
```

File: source/index.html.md [hunk 2]
```diff
@@ -451,14 +451,19 @@ curl \
...
+
+<aside>
+Wagers and Payouts will be presented back to the player using the <code>game_label</code> provided by the Round.
```
Comment (NolanK) [inline source/index.html.md]:
```suggestion
Wagers and Payouts will be presented back to the player using the <code>game_label</code> provided for the Round.
```

File: source/index.html.md [hunk 3]
```diff
@@ -450,15 +450,20 @@ curl \
 -X POST \
 -d '{
   "external_round_id": "partner-round-id-1",
-  "game_id": "ePz8u6hbncrNzFRCqLbX8e"
+  "external_game_id": "ePz8u6hbncrNzFRCqLbX8e"
```
Comment (NolanK) [inline source/index.html.md]:
```suggestion
  "external_game_id": "ePz8u6hbncrNzFRCqLbX8e",
```

File: source/index.html.md [hunk 4]
```diff
@@ -450,15 +450,20 @@ curl \
...
 ```
 `POST /users/{user_id}/rounds`
 
 This method creates a round. A round is a container used for grouping together wagers and payouts that are associated to a user. Any combination of wagers and payouts can be contained within a round, as long as the round has not been closed.
 
-One round may share many wager and payout events as long as the game_id and user_id remain the same.
+A round can consist of zero or more Wagers, and zero or more Payouts. The choice of how many Wagers and Payouts to include in a round is up to the integration partner.
+
+<aside>
+Wagers and Payouts will be presented back to the player using the <code>external_game_label</code> provided by the Round.
```
Comment (NolanK) [inline source/index.html.md]:
```suggestion
Wagers and Payouts will be presented back to the player using the <code>external_game_label</code> provided for the Round.
```

File: source/index.html.md [hunk 5]
```diff
@@ -1023,8 +1027,16 @@ A successful request returns the HTTP `200 OK` status code with the following JS
 | Parameter | Type | Example | Description |
 | ---| ---| ---| --- |
 | amount | Money | 100.00 | The user's balance to a maximum of 2 decimal places. |
-| currency | string | USD | Currency specified by an ISO-4217 3-letter code. |
+| currency | string | USD | Currency.  When relevant this will reference ISO-4217 3-letter code. |
+
+
+<aside class="comment">
+Currency code <code>XXX</code> will be used to represent free money funds.
```
Comment (dennis-mah) [inline source/index.html.md]:
```suggestion
Currency code <code>XXX</code> will be used to represent simulated currencies.
```

ficticious?

File: source/index.html.md [hunk 6]
```diff
@@ -1023,8 +1027,16 @@ A successful request returns the HTTP `200 OK` status code with the following JS
...
+</aside>
 
+<aside class="comment">
+Currencies are associated to accounts during creation of the integration.
```
Comment (NolanK) [inline source/index.html.md]: "creation of the integration" doesn't make sense to me and might need rewording.

**Review by NolanK — APPROVED** [review]
One small comment but LGTM

File: source/index.html.md (hunk 6 above)
Comment (ShawnSidoruk) [reply source/index.html.md]:
```suggestion
Possible currencies returned are setup within the Betable API during the initial phases of partner integration
```

**Pull request merged**